luthier/pom.xml

```diff
@@ -34,5 +34,11 @@
             <artifactId>fili-core</artifactId>
             <type>test-jar</type>
         </dependency>
+        <dependency>
```

Comment:
Why are we adding this dependency?

Reply:
That was an oversight, my IDE auto-added that in a commit without my notice.